src/titles.json

```diff
@@ -2,7 +2,18 @@
     {
         "name": "Friends",
         "access_key": "ridfebb9",
-        "nex_version": "1.0.0"
+        "nex_version": "1.0.0",
+        "title_ids": [
+            "0004003000008D02",
+            "0004003000009602",
+            "0004003000009F02",
+            "000400300000A702",
+            "000400300000AF02",
+            "000400300000B702",
```

Comment:
These are the title IDs of the friend applets. The 3DS uses the friends sysmodule to connect to the friend servers, so I would set the title ID of the sysmodule (0004013000003202) instead of these.

I would think a similar story happens on the Wii U, but I'm not certain about that

Reply:
On wii u all i could find was the system applet title ids for friends list (the 3 that were in there), but for 3ds i did change it to the sysmodule title id